bundles/CoreBundle/Resources/migrations/Version20180904201947.php

```diff
@@ -17,6 +18,14 @@ public function up(Schema $schema)
         $table->addColumn('workflow', 'string', ['length' => 100]);
         $table->dropPrimaryKey();
         $table->setPrimaryKey(['cid', 'ctype', 'workflowId', 'workflow']);
+
+
+        $db = Db::get();
+        try {
+            $db->query("INSERT INTO users_permission_definitions VALUES('workflow_details');");
+        } catch (\Exception $e) {
+        }
+
     }
```

Comment:
`$this->addSql('INSERT INTO users_permission_definitions VALUES('workflow_details');')`